src/gc.c

```diff
@@ -3957,6 +3957,29 @@ void jl_free_thread_gc_state(jl_ptls_t ptls)
     arraylist_free(&mq->reclaim_set);
 }
 
+void jl_spawn_gc_threads(void)
```

Comment:
Suggest calling this `jl_start_gc_threads` for symmetry with the `jl_start_threads` we already have.

Some comments on the function would be great.